### PR TITLE
exporter: add mattermost exporter

### PR DESCRIPTION
**Pull request by cpanato**
This PR adds a Mattermost exporter, it is based on the slack exporter.

thanks for this project! we use that every day :)

### CODE REVIEW

**Comment by marcosnils** [comment]
Hey @cpanato ! Have u checked https://github.com/topfreegames/fluxcloud/issues/6 ?

**Comment by cpanato** [comment]
@marcosnils saw now that. Do you want to move to that library? feel free to close this if you want

**Comment by marcosnils** [comment]
> Do you want to move to that library? feel free to close this if you want

We're not sure yet.. We're discussing options on that thread.. would love your PoV

**Comment by cpanato** [comment]
@marcosnils I like that makes it easier to add a new notification, but by a quick look it supports the only simple message and not attachments, but that can be fixed :)

File: README.md [hunk 1]
```diff
@@ -29,6 +29,10 @@ Set the following environment variables in your chosen deployment:
 * `SLACK_ICON_EMOJI`: the Slack emoji to use as the icon.
 * `SLACK_ICON_EMOJI`: the Slack emoji to use as the icon.
 * `SLACK_ICON_EMOJI`: the Slack emoji to use as the icon.
+* `MATTERMOST_URL`: the Slack [webhook URL](https://docs.mattermost.com/developer/webhooks-incoming.html) to use.
```
Comment (fdpeiter) [inline README.md]:
```suggestion
* `MATTERMOST_URL`: the Mattermost [webhook URL](https://docs.mattermost.com/developer/webhooks-incoming.html) to use.
```

File: README.md [hunk 2]
```diff
@@ -73,6 +77,12 @@ events from namespace `team-b` to `#teamb` you would set the following string:
 Set the environment variable `MSTEAMS_URL` to the URL generated on activation of an
 Incoming Webhook in a Microsoft Teams channel.
 
+## Mattermost
+
+The default exporter to use is Mattermost. To use the Mattermost exporter, set the `MATTERMOST_URL`,
```
Comment (fdpeiter) [inline README.md]: There isn't a default exporter.
```suggestion
To use the Mattermost exporter, set the `MATTERMOST_URL`,
```

**Comment by fdpeiter** [comment]
Also @cpanato do you mind adding mattermost to the helm chart as well?
https://github.com/topfreegames/fluxcloud/tree/master/chart/fluxcloud

Let me know if you need any help.

**Comment by fdpeiter** [comment]
@cpanato any updates? If not, I will proceed and close the MR.

**Comment by cpanato** [comment]
@fdpeiter sorry for the long delay :(

I will update the PR based on the review and make the other one for the charts!
i will do that this weekend :)
again sorry for the huge delay

**Comment by cpanato** [comment]
ready @fdpeiter

**Review by fdpeiter — APPROVED** [review]

**Comment by fdpeiter** [comment]
Sorry for the delay @cpanato. Merged and launched a new version.

**Comment by cpanato** [comment]
cool, thanks!!

**Pull request merged**